superset/views/annotations.py

```diff
@@ -22,9 +22,12 @@ class AnnotationModelView(SupersetModelView, DeleteMixin):  # noqa
     add_title = _('Add Annotation')
     edit_title = _('Edit Annotation')
 
-    list_columns = ['layer', 'short_descr', 'start_dttm', 'end_dttm']
+    list_columns = ['layer', 'short_descr', 'start_dttm', 'end_dttm',
+                    'annotation_metadata']
```

Comment:
I think `annotation_metadata` should be `json_metadata`, though actually we don't want the json in the list view as it may bloat the table, I think I'd just leave it out.